source/blog/2018-07-06-the-ember-times-issue-54.md

```diff
@@ -0,0 +1,131 @@
+
+Ember Twiddle, currently at `v0.15.0` is a great tool that allows creating simple Ember apps to showcase ideas, bugs, issues, demos and more. It allows using addons with the included `twiddle.json` and has a built-in generator just like Ember-CLI. There are even different keyboard layouts such as Vim, Emacs, Sublime and more.
+
+Now is a great time to check out this amazing tool!
+
+---
+
+## [Upgrade ⬆️ Your App up to 11 with Ember 3.2 🐹](https://www.emberjs.com/blog/2018/07/02/ember-3-2-released.html)
+
+The time has come. The long awaited [version 3.2 release of Ember, Ember CLI and Ember Data](https://www.emberjs.com/blog/2018/07/02/ember-3-2-released.html) is out in the wild!
+
+A little later than previously planned, this release makes up for the waiting with neat new features,
+including the handy [let helper for on the spot creation of bindings in your templates](https://emberjs.github.io/rfcs/0286-block-let-template-helper.html) or
+the ✨ testing assertion library [qunit-dom](https://github.com/simplabs/qunit-dom)
+which is now added to your v3.2+ app on your very first `ember new`.
+
+Ember Data's latest release
+[removes](https://www.emberjs.com/blog/2018/07/02/ember-3-2-released.html#toc_ember-data-feature-flag-removal-2-of-4) several **stale feature flags** -
+put into place to allow users an opt-in to experimental features which have now become outdated
+to make way for more recent developments of the addon.
+If you're relying on any of these features in your app, be sure to
+**reach out** to the Ember Data team for support with the transition [by opening an issue here](https://github.com/emberjs/data/issues).
+
+You can also check out the release blog post's **Deprecations** sections to ensure that you're app is prepared for the
+next major release. Read all about 👀 it on the Ember Blog:
+
+<div class="blog-row">
+  <a class="ember-button"
+    href="https://www.emberjs.com/blog/2018/07/02/ember-3-2-released.html"
+    target="blog"
+    aria-label="Read more about the 3.2 Ember release on the Ember blog">Read more</a>
+</div>
+
+---
+
+## [Open Source Live](https://www.youtube.com/watch?v=Dx0vfMyQJMU&feature=youtu.be)
+Now that the Guides and Deprecations have been moved into Ember apps, [emberjs.com](https://emberjs.com/) needs to be moved to an Ember app too! [Chris Manson](https://github.com/mansona) and [Melanie Sumner](https://github.com/MelSumner) recorded their latest pairing session, where they are working on adding the beta [ember-learn/ember-styleguide](https://github.com/ember-learn/ember-styleguide) to the new [ember-learn/ember-website](https://github.com/ember-learn/ember-website) (a rewrite of [emberjs/website](https://github.com/emberjs/website) in Ember, so as to make it easier to contribute to). Check out the video to learn more about Learning Team happenings and pair programming for knowledge sharing!
```

Comment:
What do you think about highlighting `latest pairing session` and / or `adding the beta` here?